content/docs/next-release-v2/deployment.md

```diff
@@ -151,46 +96,7 @@ At default settings **jaeger-collector** exposes the following ports:
 | 9411  | HTTP     | `/api/v1/spans` and `/api/v2/spans` | Accepts Zipkin spans in Thrift, JSON and Proto (disabled by default).
 | 14250 | gRPC     | n/a      | Used by **jaeger-agent** to send spans in [model.proto][] Protobuf format.
 
-## Ingester
-
-**jaeger-ingester** is a service which reads span data from Kafka topic and writes it to another storage backend (Elasticsearch or Cassandra).
-
-Port  | Protocol | Function
------ | -------  | ---
-14270 | HTTP     | admin port: health check at `/` and metrics at `/metrics`
-
-To view all exposed configuration options run the following command:
-```sh
-docker run \
-  -e SPAN_STORAGE_TYPE=cassandra \
-  jaegertracing/jaeger-ingester:{{< currentVersion >}}
-  --help
-```
-
-## Query Service & UI
-
-**jaeger-query** serves the API endpoints and a React/Javascript UI.
-The service is stateless and is typically run behind a load balancer, such as [**NGINX**](https://www.nginx.com/).
-
-At default settings the **jaeger-query** service exposes the following port(s):
-
-Port  | Protocol | Function
------ | -------  | ---
-16685 | gRPC     | Protobuf/gRPC [QueryService](https://github.com/jaegertracing/jaeger-idl/blob/master/proto/api_v2/query.proto)
-16686 | HTTP     | `/api/*` endpoints and Jaeger UI at `/`
-16687 | HTTP     | admin port: health check at `/` and metrics at `/metrics`
-
-### Minimal deployment example (Elasticsearch backend):
-```sh
-docker run -d --rm \
-  -p 16685:16685 \
-  -p 16686:16686 \
-  -p 16687:16687 \
-  -e SPAN_STORAGE_TYPE=elasticsearch \
-  -e ES_SERVER_URLS=http://<ES_SERVER_IP>:<ES_SERVER_PORT> \
-  jaegertracing/jaeger-query:{{< currentVersion >}}
-```
-
+!!! NEED TO FIX 2 sections BELOW
 ### Clock Skew Adjustment
```

Comment:
need to move elsewhere, this is not a "deployment" section. Technically it belongs to query/UI role, so could move there.

Reply:
Do you want me to put an example UI/query role here for the deployment instead? I can add the snippets.

Reply:
we need to mention the UI/query role on this page, but the actual configuration details can go to its own dedicated paged.